components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -101,17 +102,22 @@ - (void)commonMDCBottomNavigationBarInit {
     }
   }
   _maxLandscapeClusterContainerWidth = kMDCBottomNavigationBarLandscapeContainerWidth;
-  _containerView = [[UIView alloc] initWithFrame:CGRectZero];
-  _containerView.autoresizingMask =
+  _inkClippingView = [[UIView alloc] init];
+  _inkClippingView.clipsToBounds = YES;
+  _inkClippingView.autoresizingMask =
+      UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleWidth;
+  _itemsLayoutView = [[UIView alloc] initWithFrame:CGRectZero];
+  _itemsLayoutView.autoresizingMask =
       (UIViewAutoresizingFlexibleLeftMargin | UIViewAutoresizingFlexibleRightMargin);
-  _containerView.clipsToBounds = YES;
-  [self addSubview:_containerView];
+  _itemsLayoutView.clipsToBounds = NO;
+  [self addSubview:_inkClippingView];
+  [_inkClippingView addSubview:_itemsLayoutView];
 #if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunguarded-availability"
 #pragma clang diagnostic ignored "-Wtautological-pointer-compare"
   if (&UIAccessibilityTraitTabBar != NULL) {
-    _containerView.accessibilityTraits = UIAccessibilityTraitTabBar;
+    _itemsLayoutView.accessibilityTraits = UIAccessibilityTraitTabBar;
```

Comment:
_Tip:_ Should we mark the `inkClippingView` as `accessibilityElement = NO`?

Reply:
For UIView, the default value is `NO`. 

From UIAccessibility.h:

```objc
/*
 Return YES if the receiver should be exposed as an accessibility element. 
 default == NO
 default on UIKit controls == YES 
 Setting the property to YES will cause the receiver to be visible to assistive applications. 
 */
@property (nonatomic) BOOL isAccessibilityElement;
```